controllers/configmap_controller.go

```diff
@@ -258,6 +262,13 @@ func hasAssociatedInstance(node *core.Node, instances []*instances.InstanceInfo)
 	return false
 }
 
+// mapToConfigMap fulfills the MapFn type, while always returning a request to the windows-instance ConfigMap
+func (r *ConfigMapReconciler) mapToConfigMap(_ client.Object) []reconcile.Request {
```

Comment:
Correct me if I am wrong, but this function (in context of the watch) fires off a reconcile request to the ConfigMap controller upon changes to BYOH Node objects? I'm just trying to make sense of the comment :slightly_smiling_face:

Reply:
That is correct, this function always returns the windows-instance configmap, and it will only be used in the context of watching Nodes. The flow here is:
ConfigMap controller gets a node event -> non-windows non-BYOH events are filtered out by the predicate -> events are mapped to the windows-instance configmap